Packs/PrismaCloudCompute/ReleaseNotes/1_2_16.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks - Prisma Cloud Compute
+- ***Breaking Change*** Fixed an issue in **prisma-cloud-compute-vulnerabilities-impacted-resources-list** command following changes in the new released version.
```

Comment:
if there are context to argent changes that are breaking, we should consider reporting them in the release notes. let's talk about that.